### ISSUE TITLE
table border

### ISSUE THREAD

**Issue by fanfare** [ghuntley/affectedbysvbornot.com#11]
i think `<table class="affected-table table">` would benefit from `<table class="affected-table table" border=1 cellpadding=5>` make it easier to read, e.g. [before](https://i.jollo.org/ri0H-tPd.png) and [after](https://i.jollo.org/LnyfYhTR.png)

**Comment by ghuntley** [comment]
yes; please send in the pull-request to make the website more usable!

i'm knee deep in sec filings rn.
